src/Python/Tutorial/Utility/common.py

```diff
@@ -31,13 +32,19 @@
 template_global_mesh = folder_scene + "integrated.ply"
 
 
+def sorted_alphanum(file_list_ordered):
+    convert = lambda text: int(text) if text.isdigit() else text
+    alphanum_key = lambda key: [convert(c) for c in re.split('([0-9]+)', key)]
```

Comment:
Quick question. Is this script can sort the following names? Image_[time_stamp]_[frame_number].

Reply:
Yes, I have just tested in python.
`file_list_ordered = ['image_1294886362.238178_10', 'image_1294886362.238179_11', 'image_1294886362.238150_3']`

the output of `sorted_alphanum(file_list_ordered)` is :

`['image_1294886362.238150_3', 'image_1294886362.238178_10', 'image_1294886362.238179_11']`